### PR TITLE
Frontend/login signup

### PR DESCRIPTION
**Pull request by aberatcan**
- Login page API call connected
- Register page API call connected
- Page structure rearranged

### CODE REVIEW

File: application/frontend/src/api/auth.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      console.log(err);
+    });
+};
+
+export const loginRequest = (values: IregisterForm): any => {
+  axios
+    .post("/login", {
+      email: values.email,
+      password: values.password,
+      username: values.username,
```
Comment (litityum) [inline application/frontend/src/api/auth.ts:25]: I think we don't need email to login. @veyisTurgut

File: application/frontend/src/components/LoginForm.tsx [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
+import { Link, useHistory } from "react-router-dom";
 
 export const LoginForm = () => {
+  const history = useHistory();
+
   const onFinish = (values: any) => {
-    console.log("Success:", values);
+    axios
+      .post("/login", {
+        email: values.email,
```
Comment (litityum) [inline application/frontend/src/components/LoginForm.tsx:11]: No need to email to login. @veyisTurgut

File: application/frontend/src/components/LoginForm.tsx [hunk 3]
```diff
@@ -1,13 +1,34 @@
...
+        password: values.password,
+        username: values.username,
+      })
+      .then((res) => {
+        if(res && res.status === 200){
+        localStorage.setItem("token", res.data.jwt);
+        localStorage.setItem("username", res.data.userDTO.username);
```
Comment (litityum) [inline application/frontend/src/components/LoginForm.tsx:18]: It would be better if we set userid.

**Review by litityum — CHANGES_REQUESTED** [review]

File: application/frontend/src/api/auth.ts (hunk 1 above)
Comment (veyisTurgut) [reply application/frontend/src/api/auth.ts:25]: You need one of them.

File: application/frontend/src/components/LoginForm.tsx (hunk 2 above)
Comment (veyisTurgut) [reply application/frontend/src/components/LoginForm.tsx:11]: Username or email, one of them is needed. Which one does not matter.

**Comment by veyisTurgut** [comment]
I repeat it again: You do not need to delete email. It works fine with or without it. 
Users should be able to use one of them as they like.

**Review by litityum — APPROVED** [review]

**Pull request merged**